n2kMappings.js

```diff
@@ -392,8 +392,29 @@ exports.mappings = {
       return n2k.fields["Temperature Instance"] + '';
     },
     source: 'Actual Temperature'
-  }]
+    }],
 
+  //Tanks
+  '127505': [{
+      source: 'Level',
+      node: function(n2k) { 
+	return 'tanks.' + n2k.fields['Type'] + '_' + n2k.fields['Instance'] + '.currentLevel;' 
```

Comment:
Fixed that, but it still doesn't like it. Also changed to just be 'fuel' in the path because it was complaining.